packages/plugin-atoms/src/components/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const LAYOUT = {
```

Comment:
Já bych to dal dopryč... příjde mi to jako zbytečný overhead... jedná se o typy propsů a když definuješ `button` apod, tak taky nepoužíváš konstantu `Type.Submit`....

Nevidím v tom žádnou výhodu, spíš obtíž, ještě k tomu když to je otypované tak ti IDE napoví správné typy.